script.js

```diff
@@ -1,4 +1,259 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+// /*
+// Included features:
+// - Last roll
+// - Winning %
+// - Most rolled
+// - Guessing +- 1
+// */
+
+/* 
+Fn:
+input: 
+- Number of users
+
+Each submit roll 2 dices
+track score
+*/
+var totalGames = 0;
+
+class player {
+  constructor(playername, dicenumber, score) {
+    this.playername = playername;
+    this.dicenumber = dicenumber;
+    this.combinedDiceNumber = this.combinedDiceNumber;
+    this.score = score;
+  }
+}

```

Comment:
nice use of classes to create player objects! good job going beyond what was taught